docs/deprecations.rst

```diff
@@ -116,6 +116,28 @@ The use in :py:mod:`.ImageCms` of input modes and output modes that are not Pill
 image modes has been deprecated. Defaulting to "L" or "1" if the mode cannot be mapped
 is also deprecated.
 
+16-Bit RGB/BGR Rawmodes
+^^^^^^^^^^^^^^^^^^^^^^^
+
+.. deprecated:: 10.4.0
+
+The following rawmodes have been deprecated and replaced with better-named rawmodes.
+
+============  ==============
+Removed       Use instead
```

Comment:
They're not 'Removed' yet, they're 'Deprecated' - see https://pillow.readthedocs.io/en/stable/deprecations.html#imagecms-constants-and-versions-function